.github/workflows/build-graphscope-images-linux.yml

```diff
@@ -146,3 +155,355 @@ jobs:
         sudo docker push ${{ env.REGISTRY }}/graphscope/interactive-frontend:${tag}
         sudo docker push ${{ env.REGISTRY }}/graphscope/interactive-executor:${tag}
         sudo docker push ${{ env.REGISTRY }}/graphscope/learning:${tag}
+
+    - name: Clean Nightly Image
+      if: ${{ github.ref == 'refs/heads/main' && github.repository == 'alibaba/GraphScope' }}
+      run: |
+        # x86_64
+        arch=$(uname -m)
+        # docker tag: 0.15.0 -> 0.15.0a20220808
+        time=$(date "+%Y%m%d")
+        version=$(cat ${GITHUB_WORKSPACE}/VERSION)
+        tag="${version}a${time}"-${arch}
+        # clean
+        sudo docker rmi -f graphscope/coordinator:${tag} || true
+        sudo docker rmi -f graphscope/analytical:${tag} || true
+        sudo docker rmi -f graphscope/analytical-java:${tag} || true
+        sudo docker rmi -f graphscope/interactive-frontend:${tag} || true
+        sudo docker rmi -f graphscope/interactive-executor:${tag} || true
+        sudo docker rmi -f graphscope/learning:${tag} || true
+        sudo docker rmi -f graphscope/dataset:${tag} || true
+        tag=${SHORT_SHA}
+        sudo docker rmi -f graphscope/coordinator:${tag} || true
+        sudo docker rmi -f graphscope/analytical:${tag} || true
+        sudo docker rmi -f graphscope/analytical-java:${tag} || true
+        sudo docker rmi -f graphscope/interactive-frontend:${tag} || true
+        sudo docker rmi -f graphscope/interactive-executor:${tag} || true
+        sudo docker rmi -f graphscope/learning:${tag} || true
+        
+    - name: Clean Release Image
```

Comment:
we don't need to clean images during `x86` building process on `github runner` machine